pmm/pmm-2-0-documentation.groovy

```diff
@@ -0,0 +1,82 @@
+pipeline {
+    agent {
+        label 'micro-amazon'
+    }
+    parameters {
+        string(
+            defaultValue: 'git@github.com:percona/pmm-doc',
+            description: 'Repository name for build',
+            name: 'REPO_NAME')
+        string(
+            defaultValue: 'master',
```

Comment:
PMM-2.0 ?

Reply:
yes, Repo's are different for docs of both version, pmm1.0 and pmm2.0